src/Analysers/METAnalyser.cpp

```diff
@@ -36,7 +36,7 @@ void METAnalyser::analyse(const EventPtr event) {
 		// }
 
 		treeMan_->setCurrentFolder(histogramFolder_);
-		treeMan_->Fill("MET",met->et());
+		treeMan_->Fill("MET_" + prefix,met->et());
 
 		// histMan_->H1D("MET_phi")->Fill(met->phi(), weight_);
 		histMan_->H1D("MET_phi")->Fill(met->phi(), weight_);
```

Comment:
This comes under histMan_ rather than treeMan_, which I'm not looking 
after or updating...

On 24/08/2015 12:20, Luke Kreczko wrote:
>
> In src/Analysers/METAnalyser.cpp 
> <https://github.com/BristolTopGroup/AnalysisSoftware/pull/180#discussion_r37741183>:
>
> > @@ -36,7 +36,7 @@ void METAnalyser::analyse(const EventPtr event) {
> >  		// }
> >
> >  		treeMan_->setCurrentFolder(histogramFolder_);
> > -		treeMan_->Fill("MET",met->et());
> > +		treeMan_->Fill("MET_" + prefix,met->et());
> >
> >  		// histMan_->H1D("MET_phi")->Fill(met->phi(), weight_);
> >  		histMan_->H1D("MET_phi")->Fill(met->phi(), weight_);
>
> Shouldn't this histogram also use the prefix?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BristolTopGroup/AnalysisSoftware/pull/180/files#r37741183>.
>